textextraction/doc_process_toolkit.py

```diff
@@ -86,33 +91,45 @@ def pdf_to_text(doc_path, port=9998):
     return document
 
 
-def process_documents(glob_path, port=9998, skip_finished=False):
+def convert_documents(glob_path, skip_converted, port=9998):
     """
     Converts pdfs to text and uses OCR if the initial attempt fails
     """
 
     for doc_path in glob.iglob(glob_path):
-        if os.path.exists(doc_path.replace('.pdf', '.txt')) and skip_finished:
+        extension = '.%s' % doc_path.split('.')[-1].lower()
```

Comment:
What you really want here is this: https://docs.python.org/3/library/os.path.html#os.path.splitext